src/Tools/dotnet-monitor/Egress/FileSystem/FileSystemEgressExtension.cs

```diff
@@ -163,5 +168,33 @@ private static string WrapMessage(string innerMessage)
                 return Strings.ErrorMessage_EgressFileFailedGeneric;
             }
         }
+
+        // This is a temporary stop-gap that simulates options binding from configuration. To do this properly, the extension
+        // egress provider or service should pass the configuration directly to the IEgressExtension implementation and allow
+        // it to decide how to pull information out of the configuration section.
+        private static void Bind(FileSystemEgressProviderOptions options, IDictionary<string, string> configuration)
+        {
+            if (configuration.TryGetValue(nameof(FileSystemEgressProviderOptions.CopyBufferSize), out string copyBufferSizeString) &&
+                !string.IsNullOrEmpty(copyBufferSizeString))
+            {
+                try
+                {
+                    options.CopyBufferSize = (int)TypeDescriptor.GetConverter(typeof(int)).ConvertFromInvariantString(copyBufferSizeString);
+                }
+                catch
+                {
```

Comment:
Do we want this to fail to bind silently?

Reply:
Yes, configuration binding fails silently for property binding. Note the comment above the method about this being a temporary stop-gap solution until more refactoring is done in future PRs.